README.md

```diff
@@ -70,6 +70,11 @@ All variables have sane defaults set in [`defaults/main.yml`](defaults/main.yml)
 | `rocket_chat_mongodb_port` | 27017 | The TCP port to contact the MongoDB host host via |
 | `rocket_chat_mongodb_packages` | `mongodb` | The name of the MongoDB package(s) to install (differs for different distros - see `vars/`) |
 | `rocket_chat_mongodb_config_template` | [`mongod.conf.j2`](templates/mongod.conf.j2) | The `/etc/mongod.conf` template to deploy |
+| `rocket_chat_mongodb_database` | `rocketchat`| MongoDB Database to use. (Advanced use only). |
+| `rocket_chat_mongodb_options` | `` | MongoDB (Connection) options. (Advanced use only). This is the place to set connection details like use TLS. |
+| `rocket_chat_mongodb_user` |  UNSET | MongoDB user (Advanced use only). Only set this is you use this! |
```

Comment:
here and next two lines need to replace:
> Only set this is you use this!

with

> Only set this if you use this!

Reply:
Darn Typo's. fixed all 3 of them in latest commit (3268a2be2e53558e27f9034316bc764620ad9c5b)